### ISSUE TITLE
Update blueprint marketplace screen in extension

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#1278]
**Context**
- We have an external marketplace at www.pixiebrix.com which hosts publicly accessible bricks and blueprints
- In https://github.com/pixiebrix/pixiebrix-app/pull/549 we added support for including sharing information with packages (TODO: double-check this is included from the packages endpoint, not just the listing endpoint)
- We want a single place for people to explore curated public blueprints for a couple reasons: 1) less development burden of maintaining parity, 2) eliminate ambiguity of where the user should go to find public content 

**User Stories**
- As a user, I need to be able to activate pre-made blueprints made available to me by a team I'm on (in the case that they're not automatically rolled out via deployments)
- As a user, I need to be able to activate blueprints I made in the workshop
- As a user, I want to be able to quickly access the public blueprint marketplace

**Implementation Sketch**
_These changes should be behind a feature flag: `public-marketplace`_ (in the React code, see `AuthContext` for accessing the flags)

- [x] Remove "Templates" section from sidebar and related code
- [x] Modify the left side nav bar. 
	- Create a Quick Links section with external links to "Marketplace", "Community", "Documentation" that open in a new tab
	- Change existing Marketplace page title to "My Blueprints"
- [x] Modify the layout of the marketplace screen: https://balsamiq.cloud/s7ffzg5/pu105es/r2673
	- Add button to open the marketplace (to the blueprints screen)
	- Only show personal and team blueprints
	- Indicate the team name on the blueprint (or indicate if it's a personal blueprint)
	- Change Add/Added to Activate/Activated to be consistent with our language on the My Bricks page
	- Add pagination (there's a [PaginationControl](http://github.com/pixiebrix/pixiebrix-extension/blob/2bbbb7422107e9a806c0b6238fc5f0b6395f377d/src/components/pagination/Pagination.tsx#L25-L25) here)

**Out of Scope**
- Does not include the brick selection modal, e.g., in the Page Editor, Add Integration, etc.

**Comment by mnholtz** [comment]
> Only show personal and team blueprints

@twschiller it looks like the current Blueprint list in the marketplace [is coming from the Redux store](https://github.com/pixiebrix/pixiebrix-extension/blob/ea3b5e9efde6a9b27ac98c229592d196756798bf/src/options/slices.ts#L174-L174). I think I might need more information to understand how to access unactivated personal and team blueprints.

**Comment by twschiller** [comment]
- The available blueprints is coming from the server: http://github.com/pixiebrix/pixiebrix-extension/blob/2bbbb7422107e9a806c0b6238fc5f0b6395f377d/src/pages/marketplace/MarketplacePage.tsx#L123-L123. This is what will need to be filtered to only show the personal and team blueprints. This PR should have added sharing information to the data coming back from the API: https://github.com/pixiebrix/pixiebrix-app/pull/549/files#diff-bb86a3dbf84b56b405f4b9f162950fc093e703d610f9a2ea97143d8d21c25b3eR53

- You are correct that the set of currently installed blueprints is coming from the Redux store: http://github.com/pixiebrix/pixiebrix-extension/blob/2bbbb7422107e9a806c0b6238fc5f0b6395f377d/src/options/pages/MarketplacePage.tsx#L57-L57

**Comment by mnholtz** [comment]
@twschiller "organizations" in the metadata always comes back empty. What is the difference between a package `organization` and package `organizations`?

http://github.com/pixiebrix/pixiebrix-app/blob/902c699c147935751d2edae586c6c797cb408032/api/models/registry.py#L274-L289

In other words, in `get_user_sharing`:

```
    def get_user_sharing(self, user):
        return dict(
            public=self.public,
            organizations=self.organizations.filter(
                id__in=user.get_organizations()
            ).values_list("id", flat=True),
        ) 
```
self.organizations returns an empty array, so there are no organizations to filter over.

Currently blocked on this.

**Comment by twschiller** [comment]
* `self.organizations` when used on `Package` is referring to that organizations many-to-many field, which is the set of organizations the package has been shared with. If it's a public package, it won't be set
* `user.get_organizations()` is the organizations the user making the request is a member of
* `get_user_sharing` filters by user.get_organizations() so the user doesn't see UUIDs of organizations they're not a member of

The way an organization would be set is on the Share modal: 
![image](https://user-images.githubusercontent.com/1879821/133320704-575b5753-3c10-43e1-aada-797f57acdb07.png)

Or in the workshop:
![image](https://user-images.githubusercontent.com/1879821/133320729-fd9bcea3-3e19-47b5-a973-0c2f9e5d13f1.png)

**Comment by mnholtz** [comment]
> * If it's a public package, it won't be set

Ahh, this is helpful. I have been using with public packages.

**Comment by mnholtz** [comment]
@twschiller should we return "author" as data from the api? Otherwise, I will be parsing & inferring the author from the scope in package id.
